src/attr/_make.py

```diff
@@ -669,24 +669,28 @@ def _create_slots_class(self):
             if k not in tuple(self._attr_names) + ("__dict__", "__weakref__")
         }
 
-        # Traverse the MRO to check for an existing __weakref__ and
-        # __setattr__.
-        custom_setattr_inherited = False
+        # If our class doesn't have an own __setattr__ (either from the user or
+        # by us), check the bases, if one of them has an attrs-made
+        # __setattr__, that needs to be reset.
+        # XXX: This can be confused by subclassing a slotted attrs class with
+        # XXX: a non-attrs class and subclass the resulting class with an attrs
+        # XXX: class.  See `test_slotted_confused for details.  For now that's
```

Comment:
```suggestion
        # XXX: class.  See `test_slotted_confused` for details.  For now that's
```

Just noticed this unclosed backtick.

Reply:
I think this got lost in the shuffle, this still has an unclosed backtick.

Reply:
WTH, I clicked commit suggestion. 😳